nc-news/src/components/SingleArticle/SingleArticle.jsx

```diff
@@ -21,7 +22,7 @@ const SingleArticle = () => {
         <img src={currArticle.article_img_url} alt="article image" className='article-img'/>
         <p>{currArticle.body}</p>
         <section>
-            <p>Comments</p>
+            <Link to={`/articles/${article_id}/comments`}><p>Comments</p></Link>
```

Comment:
Does this have to be on it's own page? There seems to be enough room for it on the article page and I think it makes for a slightly better experience to not have to follow a link to see the comments, but not a dealbreaker either way